xclim/indicators/atmos/_temperature.py

```diff
@@ -373,7 +375,8 @@ class TempHourlyWithIndexing(ResamplingIndicatorWithIndexing):
     long_name="Total length of heat spells.",
     description="{freq} total length of heat spell events. "
     "A heat spell occurs when the {window}-day  averages of daily minimum and maximum temperatures "
-    "each exceed {thresh_tasmin} and {thresh_tasmax}.  All days of the {window}-day period are considered part of the spell.",
+    "each exceed {thresh_tasmin} and {thresh_tasmax}.  All days of the {window}-day period are considered part of the spell."
+    "Gaps of under {min_gap} days are allowed within a spell.",
```

Comment:
english? I'm not 100% but the formulation above seems weird
```suggestion
    "Gaps of less than {min_gap} days are allowed within a spell.",
```

Reply:
@Zeitsperre what do you think ?

Reply:
My official Anglo opinion: `Gaps of {min_gap} day(s) or less are allowed within a spell.` or `Gaps of fewer than {min_gap} day(s) are allowed within a spell.`.